app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java

```diff
@@ -9,7 +9,11 @@
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.preference.PreferenceManager;
+import android.util.Log;
```

Comment:
remove unnecessary import

Reply:
okay i will remove unnecessary imports and comments